docker-compose.yml

```diff
@@ -28,6 +31,62 @@ services:
       BIGCHAINDB_DATABASE_PORT: 27017
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
       BIGCHAINDB_WSSERVER_HOST: 0.0.0.0
+      BIGCHAINDB_START_TENDERMINT: 0
+      BIGCHAINDB_TENDERMINT_HOST: tendermint
+      BIGCHAINDB_TENDERMINT_PORT: 46657
     ports:
       - "9984"
-    command: bigchaindb start
+      - "46658"
+    healthcheck:
+      test: ["CMD", "curl", "http://bigchaindb:9984"]
+      interval: 3s
+      timeout: 5s
+      retries: 3
+    command: bigchaindb -l DEBUG start
+  tendermint:
+    image: tendermint/tendermint:0.13
+    volumes:
+      - ./tmdata/config.toml:/tendermint/config.toml
+    entrypoint: ''
+    ports:
+      - "46656"
+      - "46657"
+    command: bash -c "tendermint init && tendermint node"
+  bdb:
```

Comment:
whats this service doing?

Reply:
This is a proxy service to make sure BigchainDB is running, please check the description of the PR.

Reply:
do we really need a separate health check service for dev environment?